pkg/exporter/ovs.go

```diff
@@ -41,8 +41,9 @@ const (
 
 	SFlowPort = 6666
 
-	emptyUUID    = "00000000-0000-0000-0000-000000000000"
-	vSwitchdPath = "/var/run/openvswitch/ovs-vswitchd"
+	emptyUUID             = "00000000-0000-0000-0000-000000000000"
+	vSwitchdPath          = "/var/run/openvswitch/ovs-vswitchd"
+	InternalBridgeKeyword = "internal"
```

Comment:
Internal bridge has fixed bridge name, not only suffix.

Reply:
has been modified